torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -262,52 +269,79 @@ def get_aten_name(
         raise diagnostics.RuntimeErrorWithDiagnostic(diagnostic)
 
     @_beartype.beartype
+    @diagnostics.diagnose_call(
+        diagnostics.rules.find_operator_overloads_in_onnx_registry,
+        diagnostic_message_formatter=_find_operator_overloads_in_onnx_registry_disagnostic_message_formatter,
+    )
     def get_function_overloads(
         self,
         node: torch.fx.Node,
-        aten_name: str,
         diagnostic_context: diagnostics.DiagnosticContext,
-    ) -> Set[Union["onnxscript.OnnxFunction", "onnxscript.TracedOnnxFunction"]]:
+    ) -> List[registration.SymbolicFunction]:
         """Get the function overloads from the registry.
 
         Args:
             node: The node to get the function overloads for.
-            aten_name: The aten name of the node.
             diagnostic_context: The diagnostic context to use for reporting errors.
 
         Returns:
-            Set of function overloads.
+            The list contains SymbolicFunctions, starting with the default ones and
+            followed by any custom ones.
         """
-        function_group = None
 
-        if self.onnx_registry.is_registered_op(aten_name, self.opset_version):
-            function_group = self.onnx_registry.get_function_group(aten_name)
+        internal_opname: registration.OpName = self._get_aten_name(
+            node=node, diagnostic_context=diagnostic_context
+        )
+
+        # NOTE: If the ATen/Custom operators are not registered, the group will be None.
+        # And non-registerd ATen/Custom operators will trigger error in the next step.
+        function_group: Optional[List[registration.SymbolicFunction]] = None
 
-        # Fall back to overloadpacket name: eg: aten.add.Tensor -> aten::add
-        elif hasattr(
-            node.target, "overloadpacket"
-        ) and self.onnx_registry.is_registered_op(
-            node.target.overloadpacket._qualified_op_name,  # type: ignore[union-attr]
-            self.opset_version,
+        if self.onnx_registry.is_registered_op(
```

Comment:
nit: I think we can call `get_functions` directly and if result is None then we'd know it is not registered op?

Reply:
Make sense. Changed to use `get_functions` only.